src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
Again, appreciate the details.

I'm wondering though: if NewRelic isn't a logging target, why would you want to do any of this via a logging appender?

*All* Timbre error calls are then going to go through NewRelic; I can't imagine that that's going to be a common desire? Timbre users often have multiple appenders active at once. Is any application `warn`,`error`,`fatal` call anywhere in a user's application really supposed to go through NewRelic?

If you have some section of code that specifically deals with NewRelic, why not just write a function that calls `NewRelic/noticeError` in the specific cases where that makes sense?

Just trying to understand why you'd want Timbre to be involved with this at all? Apologies if I'm missing something obvious.

Thanks! :-)